bin/flash.flashair

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
```

Comment:
... OK, I had intended at start that this would have more guts to it from your other scripts to had it that way but I see it ended up being odds and ends I wrote, changing, thanks.